scripts/falco-driver-loader

```diff
@@ -113,13 +113,17 @@ get_target_id() {
 	elif [ -f "${HOST_ROOT}/etc/centos-release" ]; then
 		# Older CentOS distros
 		OS_ID=centos
-	elif [ -f "${HOST_ROOT}/etc/VERSION" ]; then
-		OS_ID=minikube
 	else
 		>&2 echo "Detected an unsupported target system, please get in touch with the Falco community"
 		exit 1
 	fi
 
+	# Overwrite the OS_ID if /etc/VERSION file is present.
+	# Not sure if there is a better way to detect minikube.
+	if [ -f "${HOST_ROOT}/etc/VERSION" ]; then
```

Comment:
From this change, i get that minikube has both /etc/VERSION and /etc/os-release, right?
But /etc/os-release took precedence.

Reply:
Yep, that's right!
Here the content of `/etc/os-release`:
```bash
$ cat /etc/os-release 
NAME=Buildroot
VERSION=2021.02.12-1-g14f2929-dirty
ID=buildroot
VERSION_ID=2021.02.12
PRETTY_NAME="Buildroot 2021.02.12"
```

Reply:
Does `uname -a` say anything about `minikube` ? Perhaps that could be the cleanest way to check for minikube!

Reply:
Unfortunately nothing related to minikube.